samples/Sentry.Samples.AzureFunctions.Worker/Program.cs

```diff
@@ -0,0 +1,15 @@
+using Microsoft.Extensions.Hosting;
+using Sentry.AzureFunctions.Worker;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults((host, builder) =>
```

Comment:
@mattjohnsonpint, notice the change here - I had to use the overload with the `HostBuilder` and `AzureFunctionsBuilder` to get access to `IConfiguration`.